### PR TITLE
[12.0] [MIG] sale_stock_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of sale_stock_operating_unit to version 12.0

### CODE REVIEW

File: sale_stock_operating_unit/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    warehouse_id = fields.Many2one(
+        comodel_name='stock.warehouse',
+        default=_default_warehouse_id)
+
+    @api.onchange('team_id')
+    def onchange_team_id(self):
+        super(SaleOrder, self).onchange_team_id()
+        if self.team_id and self.team_id.operating_unit_id:
+            warehouses = self.env['stock.warehouse'].search(
```
Comment (nikul-serpentcs) [inline sale_stock_operating_unit/models/sale_order.py:31]: Use `limit=1`

File: sale_stock_operating_unit/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                [('operating_unit_id', '=',
+                  self.team_id.operating_unit_id.id)])
+            if warehouses:
+                self.warehouse_id = warehouses[0]
+
+    @api.onchange('operating_unit_id')
+    def onchange_operating_unit_id(self):
+        if self.operating_unit_id:
+            warehouses = self.env['stock.warehouse'].search(
```
Comment (nikul-serpentcs) [inline sale_stock_operating_unit/models/sale_order.py:41]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

File: sale_stock_operating_unit/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# © 2015-19 Eficent Business and IT Consulting Services S.L. -
```
Comment (nikul-serpentcs) [inline sale_stock_operating_unit/__manifest__.py]: © - > Copyright in all file

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor change

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test + code

**Comment by AaronHForgeFlow** [comment]
Hi @nikul-serpentcs could you update your review please? Thank you.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**